ecosystem/typescript/sdk/src/token_client.ts

```diff
@@ -321,7 +397,7 @@ export class TokenClient {
    */
   async getTokenForAccount(account: MaybeHexString, tokenId: TokenTypes.TokenId): Promise<TokenTypes.Token> {
     const tokenStore: { type: Gen.MoveStructTag; data: any } = await this.aptosClient.getAccountResource(
-      account,
+      account instanceof HexString ? account.hex() : account,
```

Comment:
why is this? aptosClient handles this already by ensuring hex string is used.

Reply:
Well without this it was failing if I used `account` instead of `account.hex()` when calling this function. Or perhaps it was the other one for which that was true, in which case this is just for uniformity. I'll investigate AptosClient.

Reply:
interesting. please investigate.